### PR TITLE
CODEOWNERS: remove chrismwendt

### PR DESCRIPTION
**Pull request by chrismwendt**
Not sure who to replace myself with cc @nicksnyder

### CODE REVIEW

**Comment by unknwon** [comment]
😭

**Comment by chrismwendt** [comment]
I think this is the free part of the extension registry. There's another part under `enterprise/`.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -49,7 +49,7 @@
 /internal/processrestart @slimsag @keegancsmith
 /internal/honey @keegancsmith
 /internal/ratelimit @beyang
-/internal/registry @chrismwendt
+/internal/registry @efritz
```
Comment (nicksnyder) [inline .github/CODEOWNERS]:
```suggestion
/internal/registry @sourcegraph/web
```

**Comment by mrnugget** [comment]
GitHub doesn't allow "approve but also disapprove 😭"

**Pull request merged**